lib/fathead/gradle_docs/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Added this for when working with just this folder. Seems the top level doesn't work when i try to push from this folder.